src/braket/jobs/hybrid_job.py

```diff
@@ -203,6 +208,27 @@ def job_wrapper(*args, **kwargs):
     return _hybrid_job
 
 
+def _validate_python_version(aws_session: AwsSession, image_uri: str | None):
+    if image_uri:
+        print(
+            "Skipping python version validation, make sure versions match "
+            "between local environment and container."
+        )
+    else:
+        image_uri = retrieve_image(Framework.BASE, aws_session.region)
```

Comment:
Why is just the base framework evaluated?

Reply:
If no image_uri is provided, then the base image is used. If an image_uri is provided, then we skip validation. I can update this to instead check if the provided image_uri matches one of our built in ones